### DISCUSSION TITLE
Spelling error for Sony 20-70mm lens name

### DISCUSSION THREAD

**Discussion by jsachs00** [lensfun/lensfun#2381] [General]
In the file mil-sony.xml, there is an entry:

    <lens>
        <maker>Sony</maker>
        <model>FE 20-70mm f/4 G</model>
        <mount>Sony E</mount>
        <cropfactor>1</cropfactor>

The <model> does not correspond to the lens model returned in EXIF data. It should be:

        <model>FE 20-70mm F4 G</model>

**Comment by sarunasb** [comment]
Lensfun is not case sensitive. It also ingnores `/`.
